src/main/java/com/dumbdogdiner/stickyapi/bungeecord/util/SoundUtil.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bungeecord.util;
+
+import com.dumbdogdiner.stickyapi.StickyAPI;
+import com.dumbdogdiner.stickyapi.bungeecord.packet.SoundPacket;
+import com.dumbdogdiner.stickyapi.common.util.NotificationType;
+
+import org.jetbrains.annotations.NotNull;
+
+import net.md_5.bungee.api.CommandSender;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+
+/**
+ * Handles the playing of cute fox noises for notification purposes.
+ */
+public class SoundUtil {
+    /**
+     * Check if the parsed sender can receive sounds. Returns true if valid.
+     * 
+     * @param sender {@link org.bukkit.command.CommandSender} The sender to validate
+     * @return {@link java.lang.Boolean}
+     */
+    private static Boolean validate(CommandSender sender) {
+        return sender instanceof ProxiedPlayer;
+    }
+
+    /**
+     * Queue a specific sound to be run at a later date.
+     * 
+     * @param player The player to play the sound to
+     * @param sound  The sound to play
+     * @param volume The volume of the sound
+     * @param pitch  The pitch of the sound
+     * @param delay  The time to delay
+     */
+    public static void queueSound(@NotNull ProxiedPlayer player, @NotNull Sound sound, float volume,
+            float pitch, @NotNull Long delay) {
+        StickyAPI.getPool().submit(() -> {
+            try {
+                Thread.sleep(delay);
+                // So, since we don't have the player's position (yet...) So, we have to play this at the center of the world
+                // at the max volume... Yes, this distorts the sound, please hold while I map more packets
+                // so I can obtain the player's position and play the sound that way... -zach
+                player.unsafe().sendPacket(new SoundPacket(sound.getId(), 0, 0, 255, 0, Float.MAX_VALUE, pitch));
```

Comment:
I'm surprised this works

Reply:
This.... doesn't fully work but its a decent base thing.